lib/index.js

```diff
@@ -13,8 +13,10 @@ exports.register = (server, options, next) => {
   server.ext('onPreResponse', (request, reply) => {
 
     const settings = request.route.settings.plugins.totalCount;
-    const approximateCountValid = !Joi.validate(request, ApproximateCountValidator, { allowUnknown: true }).error;
-    const totalCountValid = !Joi.validate(request, TotalCountValidator, { allowUnknown: true }).error;
+    const includeApproximate = !settings || !settings.include || settings.include.indexOf('approximate') !== -1;
```

Comment:
What do you think of writing the condition as following (I think it's a bit easier to understand this way but it might just be me)?
```js
const includeApproximate = !(settings && settings.include) || settings.include.indexOf('approximate') !== -1;
const includeTotal = !(settings && settings.include) || settings.include.indexOf('total') !== -1;
 ```